diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter4/NullChecksRule.kt

```diff
@@ -137,6 +137,7 @@ class NullChecksRule(configRules: List<RulesConfig>) : DiktatRule(
             if (parent != null && parent.elementType == VALUE_ARGUMENT) {
                 val grandParent = parent.treeParent
                 if (grandParent != null && grandParent.elementType == VALUE_ARGUMENT_LIST && isRequireFun(grandParent.treePrev)) {
+                    @Suppress("COLLAPSE_IF_STATEMENTS")
```

Comment:
This should be placed on the outer `if`

Reply:
For now it doesn't work https://github.com/cqfn/diKTat/issues/824